ert3/stats/_stats.py

```diff
@@ -1,19 +1,26 @@
-from typing import Optional, Callable, cast
+from typing import Optional, Callable, Union, cast
 
 import numpy as np
 import scipy.stats
+import typing_extensions
 
 import ert
 
+ndarrytype: typing_extensions.TypeAlias = (
```

Comment:
Is this an "intentional typo"? `ndarrytype` vs `ndarraytype`

Reply:
yes it was a typo, but it wouldn't work anyway `numpy.typing` doesn't come into play until numpy `1.20`